packages/superset-ui-legacy-plugin-chart-heatmap/src/Heatmap.js

```diff
@@ -109,8 +109,8 @@ function Heatmap(element, props) {
 
     for (let i = 0; i < records.length; i++) {
       const datum = records[i];
-      longestX = Math.max(longestX, datum.x.toString().length || 1);
-      longestY = Math.max(longestY, datum.y.toString().length || 1);
+      longestX = Math.max(longestX, datum.x ? datum.x.toString().length || 1 : 1);
```

Comment:
not sure which is more readable, but this statement would only require specifying 1 once:
`(datum.x && datum.x.toString().length) || 1`

Reply:
I like that better thanks